src/main/java/wolox/gmjavatraining/models/Book.java

```diff
@@ -0,0 +1,54 @@
+package wolox.gmjavatraining.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Getter @Setter
+    @Column
+    private String genre;
+
+    @Getter @Setter
+    @Column(nullable = false)
```

Comment:
When the only thing you want is to specify a column as non-nullable, use the @NotNull annotation from package `javax.validation` so you can also skip the @Column annotation for these cases.